source/advocacy.js

```diff
@@ -0,0 +1,66 @@
+let javascriptLoader = {
+  initialize: function () {
+    this.defer(this.loadOptInMonster, 'jQuery')
+
+    this.defer(this.loadCrossDomain, 'ga')
+  },
+
+  defer: function (a, d) {
+    let b = window[d]
+    b && 'function' === typeof b ? a() : setTimeout(function () {
+      javascriptLoader.defer(a, d)
+    }, 50)
+  },
+
+  loadOptInMonster: function () {
+    let optInMonsterLoader = {
+      initialize: function () {
+        this.addScript()
+
+        this.addOptInMonsterAccessibility()
+      },
+
+      addScript: function () {
+        let script = document.createElement('script')
+
+        script.setAttribute('type', 'text/javascript')
+        script.setAttribute('src', 'https://a.optmnstr.com/app/js/api.min.js')
+        script.setAttribute('data-account', '1044')
+        script.setAttribute('data-user', '12468')
+        script.setAttribute('async', '')
+
+        document.body.appendChild(script)
+      },
+
+      addOptInMonsterAccessibility: function () {
+        if (window.jQuery) {
+          document.addEventListener('om.Campaign.load', function (event) {
+            let campaignDiv = jQuery('#om-' + event.detail.Campaign.id)
+
+            campaignDiv
+              .attr('aria-label', 'Promotional')
+              .attr('role', 'complementary')
+              .find('.boston-CloseButton').attr('aria-label', 'Close Promotional region')
```

Comment:
Wonder if the actual CSS class names are wrapped in the Campaign object. Would help to avoid having to get the particular campaign's custom class names beforehand.

Reply:
This is also assuming that they'll always use the same class names.

Reply:
Good suggestion. I updated it so the CSS name is set from the event.